.github/workflows/ci_compliance.yml

```diff
@@ -29,3 +33,31 @@ jobs:
           aws-s3-bucket-name: s2n-tls-ci-artifacts
           aws-s3-region: us-west-2
           cdn: https://d3fqnyekunr9xg.cloudfront.net
+
+      # In order to accurately check if there are any uncommitted changes, cleanup any
+      # intermediate artifacts generated during the duvet run.
```

Comment:
I would be more specific in this comment-- "intermediate artifacts" is pretty vague. Your [PR comment](https://github.com/aws/s2n-tls/pull/4205#discussion_r1333335116) was more helpful.

Reply:
https://github.com/aws/s2n-tls/pull/4205/commits/a110c4311d09ffb578801a7bc038dcd00667221b